pkg/syncer/metadatasyncer.go

```diff
@@ -732,6 +746,37 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	return nil
 }
 
+// initCnsVolumeOperationRequestCRInformer creates and starts an informer for CnsVolumeOperationRequest custom resource.
+func initCnsVolumeOperationRequestCRInformer(ctx context.Context, cfg *restclient.Config) error {
+	log := logger.GetLogger(ctx)
+	// Create an informer for CnsVolumeOperationRequest instances.
+	dynInformer, err := k8s.GetDynamicInformer(ctx, cnsvolumeoperationrequestv1alpha1.SchemeGroupVersion.Group,
+		cnsvolumeoperationrequestv1alpha1.SchemeGroupVersion.Version, cnsvolumeoperationrequest.CRDPlural,
+		metav1.NamespaceAll, cfg, true)
+	if err != nil {
+		return logger.LogNewErrorf(log, "failed to create dynamic informer for %s CR. Error: %+v",
+			cnsvolumeoperationrequest.CRDSingular, err)
+	}
+	cnsvolumeoperationrequestInformer := dynInformer.Informer()
+	_, err = cnsvolumeoperationrequestInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: nil,
+		UpdateFunc: func(oldObj interface{}, newObj interface{}) {
+			cnsvolumeoperationrequestCRUpdated(oldObj, newObj)
+		},
+		DeleteFunc: nil,
```

Comment:
we will also need to handle delete event called from cleanup function to decrease the "reserved" value from StoragePolicyUsage CR in case of volume create failure, right ?

Reply:
Added